spec/support/common_client_stubs.rb

```diff
@@ -0,0 +1,39 @@
+shared_context :stub_api_following_standards do
+  let(:response) do
+    {
+        'data' => data,
+        'page' => 1,
+        'page_size' => 1,
+        'total' => 1
```

Comment:
Not sure why my comment got hidden, but why not have these two be based on `data.length`?